tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -637,6 +654,18 @@ static function () {
 				[ 'amp-story-player' ],
 			],
 
+			'amp_google_assistant_assistjs'                => [
+				// Note: the first line is commented out because it is broken in AMP currently.
```

Comment:
I suppose the only time this will be uncommented is when there is a future spec update that adds support for `amp-google-assistant-assistjs-config > script`.

Reply:
Yes